ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -1,76 +1,410 @@
+            </TextColumn>}
+            {currentStep !== 0 && currentStep !== 4 && <TextColumn>
+              <div style={styles.textNormal}>Submit your information below to receive billing and additional information from
+                a Google billing partner representative.</div>
+            </TextColumn>}
           </FlexColumn>
-        </ModalFooter>
+        </ModalBody>
+        <hr style={{
+          width: '100%', backgroundColor: '#979797', borderWidth: '0px', height: '1px',
+          marginTop: '0.5rem',
+          marginBottom: '0.5rem'
+        }}/>
 
-        <FontAwesomeIcon
-          icon={faTimes}
-          size='lg'
-          style={{
-            color: colors.accent,
-            position: 'absolute',
-            top: '1rem',
-            right: '1rem',
-            cursor: 'pointer'
-          }}
-          onClick={() => this.props.onClose()}
-        />
-      </Modal>;
-    }
-  }
-);
+    {currentStep === 0 && <ModalFooter data-test-id='step-0-modal' style={{marginTop: 0, justifyContent: 'flex-start'}}>
+      <FlexRow style={{justifyContent: 'space-evenly'}}>
+        <FlexColumn>
+          <TextColumn>
+            <p style={styles.textHeader}>Familiar with setting up a Google Cloud Platform account?</p>
+            <p style={styles.textNormal}>Manually set up an account in <br/> Google Cloud Platform.</p>
+          </TextColumn>
+          <Button type='primary'
+                  style={{marginTop: '0.5rem', fontWeight: 500, fontSize: '14px', height: '39px', width: '188px'}}
+                  onClick={() => window.open(supportUrls.createBillingAccount, '_blank')}>
+            Read Directions
+          </Button>
+        </FlexColumn>
+        <FlexColumn>
+          <TextColumn>
+            <p style={styles.textHeader}>Let a Google billing partner create the account for you.</p>
+            <p style={styles.textNormal}>A representative will help you set up <br/>your billing account.</p>
+          </TextColumn>
+          <Button data-test-id='use-billing-partner-button'
+              type='primary'
+                  style={{marginTop: '0.5rem', fontWeight: 500, fontSize: '14px', height: '39px', width: '220px'}}
+                  onClick={() => {setCurrentStep(1); }}>
+            USE A BILLING PARTNER
+          </Button>
+        </FlexColumn>
+      </FlexRow>
+    </ModalFooter>}
+    {currentStep === 1 && <ModalFooter data-test-id='step-1-modal' style={{marginTop: 0, justifyContent: 'flex-start'}}>
+      <FlexColumn style={{justifyContent: 'space-evenly', width: '37rem'}}>
+        <div style={styles.textHeader}>Your Information</div>
+        <FlexRow style={{marginTop: '20px'}}>
+          <FlexColumn style={styles.textNormal}>
+            Your name
+            <TextInput
+                data-test-id='user-full-name'
+                style={styles.textInput}
+                disabled={true}
+                onChange={(v) => this.setState({userFullName: v})}
```

Comment:
hm, do we need `onChange` if this is always disabled? Does it set the initial value?